packages/react/src/components/FluidForm/FluidForm.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import { settings } from 'carbon-components';
+import Form from '../Form';
+import { FormContext } from './FormContext';
+
+const { prefix } = settings;
+
+function FluidForm({ className, children, ...other }) {
```

Comment:
Do we think people may want to attach a `ref` to a form?? Wasn't sure if we do it for `Form` or not 🤔

Reply:
hmm not sure. Its not done in `Form` though